dsl_parser/tests/test_get_capability.py

```diff
@@ -216,7 +216,6 @@ def test_get_capability_not_list(self):
         self._assert_parsing_fails(
             yaml,
             message="`get_capability` function argument should be a list. "
-            "Instead it is a <type 'str'> with the value: i_should_be_a_list."
```

Comment:
just, ugh, screw those error messages. I'll live with the slightly less tightly coupled asserts.